### DISCUSSION TITLE
State entities Ns panel pro

### DISCUSSION THREAD

**Discussion by Bertsl** [seaky/nspanel_pro_tools_apk#72] [Q&A]
I would like to use the entities concerning the power supply. Are these entities really functional? When I deactivate the power supply to the NS panel, I have no change of state... Thank you for your help

![entities exvb 30](https://github.com/seaky/nspanel_pro_tools_apk/assets/130467609/cf9f3a21-d709-4493-8685-19225158c632)

**Comment by bobzer** [comment]
I'm curious to understand what you could do with this ?
there is no battery inside the nspanel so AFAIK you'll always get those value

**Reply by Bertsl** [reply]
yes correct in terms of the battery it is absolutely useless, but I still thought I would have an indication of the absence of power to create an alarm (with sensors ''charging '' or ''ac'') . But I don't think I'll find anything about it. THANKS

**Reply by bobzer** [reply]
I guess it could works, anyway i guess it's either charging or unavailable

**Reply by Bertsl** [reply]
There is no change of state or transition to unavailable. It doesn't matter, I'll look for another solution. Thank you for your prompt response.
